ssl-impl/src/main/java/io/crate/protocols/postgres/ssl/SslConfiguration.java

```diff
@@ -249,7 +249,12 @@ static KeyStoreSettings loadKeyStore(Settings settings) throws KeyStoreException
     }
 
     private enum StoreType {
-        trust,
-        key
+        TRUST,
+        KEY;
+
+        @Override
+        public String toString() {
+            return this.name().toLowerCase();
```

Comment:
please always set an explicit locale to lower/uppercase calls. Otherwise with some locales this can lead to a strange behaviour, depending on how these names are used